test/stack_test.c

```diff
@@ -45,6 +45,8 @@ DEFINE_SPECIAL_STACK_OF_CONST(uchar, unsigned char)
 DEFINE_STACK_OF(SS)
 DEFINE_STACK_OF_CONST(SU)
 
+IMPLEMENT_COMPARISONS(char, char, "%c")
```

Comment:
I guess the question is do we want all of these test functions defined in testutil.c and used everywhere or do we want them in testutil.h so tests can create individual instances of them?

At the moment there is the flexibility to do either (although local versions aren't static).  What is ideal here?